operator/apis/loki/v1/lokistack_types.go

```diff
@@ -632,6 +632,14 @@ type QueryLimitSpec struct {
 
 // IngestionLimitSpec defines the limits applied at the ingestion path.
 type IngestionLimitSpec struct {
+	// DesiredRate defines the desired ingestion rate per second that LokiStack should
+	// target applyfing automatic stream sharding. Units Bytes.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:number",displayName="Desired Ingestion Rate (in Bytes)"
+	DesiredRate int32 `json:"desiredRate,omitempty"`
```

Comment:
I will stick with MB and an `int32` to keep all knobs similar. On the rendering side we have many discrepancies because of the Loki Config format, some accept bytes (i.e. `desired_rate`) while other accept megabytes w/o a suffix (i.e. `ingestion_rate_mb`) and others with a suffix (`per_stream_rate_limit`).

We can save the overhead of `resource.Quantity` for another day, e.g. LokiStack v2. ;)